modules/nf-core/barrnap/main.nf

```diff
@@ -46,7 +46,7 @@ process BARRNAP {
     def prefix = task.ext.prefix ?: "${meta.id}"
 
     """
-    touch ${prefix}.gff
+    touch ${prefix}_${db}.gff
```

Comment:
Does this see the  `db         = dbname ? "${dbname}" : 'bac'` defined above? I suspect you'll need this here too.

Reply:
Updated the PR.